src/dataset-builder/TestConstants.ts

```diff
@@ -127,33 +127,33 @@ export const dummyDatasetModel = (): DatasetModel => ({
 
 const dummyConcepts = [
   // IDs must be unique.
-  { id: 100, name: 'Condition', count: 100, hasChildren: true },
-  { id: 101, name: 'Clinical Finding', count: 100, hasChildren: true },
-  { id: 102, name: 'Disease', count: 100, hasChildren: true },
-  { id: 103, name: 'Disorder by body site', count: 100, hasChildren: false },
-  { id: 104, name: 'Inflammatory disorder', count: 100, hasChildren: false },
-  { id: 105, name: 'Degenerative disorder', count: 100, hasChildren: false },
-  { id: 106, name: 'Metabolic disease', count: 100, hasChildren: false },
-  { id: 107, name: 'Finding by site', count: 100, hasChildren: false },
-  { id: 108, name: 'Neurological finding', count: 100, hasChildren: false },
-  { id: 200, name: 'Procedure', count: 100, hasChildren: true },
-  { id: 201, name: 'Procedure', count: 100, hasChildren: true },
-  { id: 202, name: 'Surgery', count: 100, hasChildren: false },
-  { id: 203, name: 'Heart Surgery', count: 100, hasChildren: false },
-  { id: 204, name: 'Cancer Surgery', count: 100, hasChildren: false },
-  { id: 300, name: 'Observation', count: 100, hasChildren: true },
-  { id: 301, name: 'Blood Pressure', count: 100, hasChildren: false },
-  { id: 302, name: 'Weight', count: 100, hasChildren: false },
-  { id: 303, name: 'Height', count: 100, hasChildren: false },
-  { id: 400, name: 'Carcinoma of lung parenchyma', count: 100, hasChildren: true },
-  { id: 401, name: 'Squamous cell carcinoma of lung', count: 100, hasChildren: true },
-  { id: 402, name: 'Non-small cell lung cancer', count: 100, hasChildren: true },
-  { id: 403, name: 'Epidermal growth factor receptor negative ...', count: 100, hasChildren: false },
-  { id: 404, name: 'Non-small cell lung cancer with mutation in epidermal..', count: 100, hasChildren: true },
-  { id: 405, name: 'Non-small cell cancer of lung biopsy..', count: 100, hasChildren: false },
-  { id: 406, name: 'Non-small cell cancer of lung lymph node..', count: 100, hasChildren: false },
-  { id: 407, name: 'Small cell lung cancer', count: 100, hasChildren: true },
-  { id: 408, name: 'Lung Parenchcyma', count: 100, hasChildren: false },
+  { id: 100, name: 'Condition', count: 100, code: '0', hasChildren: true },
```

Comment:
Even though this is test data, if we have to provide codes for each, why not use a different code for each one? It might be helpful when writing a test later.

Reply:
Ok sounds good!